tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -65,6 +73,40 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		if (DO_GOFMT && !fileName.startsWith(".") /* criterion taken from gofmt*/ && fileName.endsWith(".go")) {
```

Comment:
What is the not-starts-with-period criterion? `echo package foo > .foo.go && gofmt -e -l -s -w .foo.go` seems to work.

    /* criterion taken from gofmt*/

Missing space between `gofmt` and `*/`